website/docs/docs/dbt-versions/release-notes/72-Feb-2024/ide-github.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "New: Edit in primary branch in IDE"
+description: "February 2024: Edit access to the primary GitHub branch while in the IDE. This was previously a read-only feature."
```

Comment:
hey @naomiwritescode is this only for github or for all git providers? if it's for all git providers, i think we can leave out github since we support other git providers. ```suggestion
description: "February 2024: You can now edit directly on your primary git branch and commit those changes to a new branch when ready. This was previously a read-only feature."
```

Reply:
Made a separate update